src/elements/stepper/stepper.tsx

```diff
@@ -0,0 +1,22 @@
+interface Stepper {
+  value: number
+  label: string
+}
+
+interface IStepperProps {
+  currentStep: number
+  steps: Stepper[]
+}
+
+export const Stepper = ({ currentStep, steps }: IStepperProps) => {
+  return (
+    <div>
+      <>
+        <div>
+          <h3>Step {steps[currentStep]!.value}</h3>
```

Comment:
Let's pass this one as a prop. It could be `title` prop

Reply:
@jiyeol-lee uhm,, bro like it?

Reply:
Nope, it's not `children`. Let's name it `title`

Reply:
uhm,, is it right?

Reply:
Yes